raspberrypi-c-on-device.js

```diff
@@ -0,0 +1,167 @@
+﻿/*
+* Gulp Common - Microsoft Sample Code - Copyright (c) 2016 - Licensed MIT
+*/
+'use strict';
+
+var args = require('get-gulp-args')();
+
+var all;
+
+function initTasks(gulp, options) {
+  all = require('./all.js')(options);
+
+  var config = all.getConfig();
+  var targetFolder = config.project_folder ? config.project_folder : '.';
+  var startFile = config.start_file ? config.start_file : 'app';
+
+  // stick config into gulp object
+  gulp.config = config;
+
+  if (typeof all.gulpTaskBI === 'function') {
+    all.gulpTaskBI(gulp, 'c', 'RaspberryPi', ((options && options.appName) ? options.appName : 'unknown'));
+  }
+
+  var runSequence = require('run-sequence').use(gulp);
+
+  gulp.task('init', 'Initializes sample', function (cb) {
+
+    if (options.configPostfix && options.configTemplate) {
+      all.updateGlobalConfig(options.configPostfix, options.configTemplate);
+    }
+
+    cb();
+  })
+
+  gulp.task('rpi-clone-azure-sdk', false, function(cb) {
+    all.sshExecCmds([ "git clone https://github.com/Azure/azure-iot-sdks.git",
```

Comment:
Do we need to clone the repo to home folder specifically? That is to add the '~' at the end of the git clone command.